File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size, i, j, k, l;

    do
    {
        size = get_int("Height: ");
    }
    while (size < 1 || size > 8);

    for (i = 1; i <= size; i++)
    {
        for (j = i; j < size; j++)
        {
            printf(" ");
        }

        for (k = 1; k <= i; k++)
        {
            printf("#");
        }

        printf(" ");

        for (l = k; l < k; l++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: cs50x/PSET_6/sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if 0 < height <= 8:
            break
    draw(height)


def draw(h):
    for i in range(h):
        for j in range(h - (i+1)):
            print(" ", end="")
        for k in range(i + 1):
            print("#", end="")
        print()


main()
